ci-operator/promotions/README.md

```diff
@@ -0,0 +1,46 @@
+The directory describes the contents of all published images generated by ci-operator jobs after code is merged to a repository branch under CI control.
```

Comment:
If this is going to mostly result in people messing with `ci-operator` configuration, I'd prefer this to be a help page or (sub-page of the ci-operator overview) on the step registry website. Ideally we're getting to a place where someone has a one-stop-shop for everything from "I'm onboarding a repo, what does that mean" to a full integration with CI

Reply:
I.e. including our processes?  That's fine, I just wanted to have it written out to link to the example.

Reply:
Yes.